### PR TITLE
Update user documentation

### PR DESCRIPTION
**Pull request by a-breid-ericsson**
### Applicable Issues
Some spelling issues and documentation focused on submit.

### Description of the Change
Extended user documentation.

### Sign-off

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

Signed-off-by:

### CODE REVIEW

File: src/main/resources/Documentation/about.md [hunk 1]
```diff
@@ -1,41 +1,55 @@
-TODO: Add Plugin introduction
+Eiffel Gerrit plugin is may be used to send Eiffel events when changes occurs in in a project in Gerrit.
```
Comment (e-pettersson-ericsson) [inline src/main/resources/Documentation/about.md]: one of is/may can be removed from this sentence :)

File: src/main/resources/Documentation/about.md [hunk 2]
```diff
@@ -1,41 +1,55 @@
...
+This plugin simplifies communication in continuous integration and delivery pipelines and helps keeping
+source code management traceability.
 
 For more Eiffel information follow this link:
 <https://github.com/eiffel-community/eiffel-gerrit-plugin>
```
Comment (e-pettersson-ericsson) [inline src/main/resources/Documentation/about.md]: This link should point to the protocol right?

File: src/main/resources/Documentation/about.md [hunk 3]
```diff
@@ -1,41 +1,55 @@
...
 
+#####Eiffel Events:
+
+Version 0.0.1
```
Comment (e-pettersson-ericsson) [inline src/main/resources/Documentation/about.md]: We might want to consider linking to git tags/releases in GitHub later on when they exist.

File: src/main/resources/Documentation/about.md [hunk 4]
```diff
@@ -1,41 +1,55 @@
...
+
 Parameters
 ----------
 
 Project Configuration:
 
 eiffel-integration.enabled
-:   Plugin state. if set to `true`, send message Eiffel message for every commit submitted for this repository.
-    Plugin considered disabled if this option is not defined or set to `false`.
+:   If set to `true` the plugin will send eiffel events.
```
Comment (e-pettersson-ericsson) [inline src/main/resources/Documentation/about.md]: capital letter for Eiffel

File: src/main/resources/Documentation/about.md [hunk 5]
```diff
@@ -1,41 +1,55 @@
...
+    Is set to `false` or is not defined no eiffel events will be sent.
 
 eiffel-integration.filter
-:   Allow users to defined a white list of branches, i.e. messages will be sent only for a commits submitted on specified branches, all other submits will be ignored.
-    Accepts Java regex's separated by spaces. If not defined, send messages for commits submitted on all branches.
+:   Allow users to defined a white list of branches, i.e. messages will be sent only for source change on
```
Comment (e-pettersson-ericsson) [inline src/main/resources/Documentation/about.md]: defined --> define

File: src/main/resources/Documentation/about.md [hunk 6]
```diff
@@ -1,41 +1,55 @@
...
+    specified branches. Source Change to any other branch will be ignored. Accepts full branch name and or
+    regex separated by spaces. If not defined, messages will be sent for source changes in all branches.
+    See Example configuration where release branch may be triggered on release-(Any version).
```
Comment (e-pettersson-ericsson) [inline src/main/resources/Documentation/about.md]: Example --> example

File: src/main/resources/Documentation/about.md [hunk 7]
```diff
@@ -1,41 +1,55 @@
...
 
 eiffel-integration.flow-context
 :   Optional, but expects a UUID of a EiffelFlowContextDefinedEvent if provided.
-    Can accept List of UUID's seperated by `,` that tell which flow it will be part of. (More about Flow Context: <https://github.com/eiffel-community/eiffel/blob/master/eiffel-vocabulary/EiffelSourceChangeCreatedEvent.md#flow_context>) 
+    Can accept List of UUID's seperated by `,` that tell which flow it will be part of. (More about Flow Context:
```
Comment (e-pettersson-ericsson) [inline src/main/resources/Documentation/about.md]: seperated  --> separated 
tell --> tells
"... which flow it will be part of."  it in this case is the Eiffel event right? Maybe clarify that.

File: src/main/resources/Documentation/about.md [hunk 8]
```diff
@@ -1,41 +1,55 @@
...
 Parameters
 ----------
 
 Project Configuration:
 
 eiffel-integration.enabled
-:   Plugin state. if set to `true`, send message Eiffel message for every commit submitted for this repository.
-    Plugin considered disabled if this option is not defined or set to `false`.
+:   If set to `true` the plugin will send eiffel events.
+    Is set to `false` or is not defined no eiffel events will be sent.
```
Comment (Christoffer-Cortes) [inline src/main/resources/Documentation/about.md]: 'If set to `false` or if not defined' and also 'Eiffel'

File: src/main/resources/Documentation/about.md (hunk 1 above)
Comment (Christoffer-Cortes) [reply src/main/resources/Documentation/about.md]: 'occur' can be in singular.

Comment (a-breid-ericsson) [reply src/main/resources/Documentation/about.md]: Done in next commit.

File: src/main/resources/Documentation/about.md (hunk 2 above)
Comment (a-breid-ericsson) [reply src/main/resources/Documentation/about.md]: This is eiffel-gerrit plugin documentation, and I guess the link should go to the wiki later on. The wiki may link to the protocol.

File: src/main/resources/Documentation/about.md (hunk 3 above)
Comment (a-breid-ericsson) [reply src/main/resources/Documentation/about.md]: Yes, and most likely this will render a 1.0.0 release and not 0.0.1.

File: src/main/resources/Documentation/about.md (hunk 4 above)
Comment (a-breid-ericsson) [reply src/main/resources/Documentation/about.md]: Done in next commit.

File: src/main/resources/Documentation/about.md (hunk 5 above)
Comment (a-breid-ericsson) [reply src/main/resources/Documentation/about.md]: Done in next commit.

File: src/main/resources/Documentation/about.md (hunk 6 above)
Comment (a-breid-ericsson) [reply src/main/resources/Documentation/about.md]: Done in next commit.

File: src/main/resources/Documentation/about.md (hunk 7 above)
Comment (a-breid-ericsson) [reply src/main/resources/Documentation/about.md]: Done in next commit.

File: src/main/resources/Documentation/about.md (hunk 8 above)
Comment (a-breid-ericsson) [reply src/main/resources/Documentation/about.md]: Done in next commit.

File: src/main/resources/Documentation/about.md (hunk 2 above)
Comment (e-pettersson-ericsson) [reply src/main/resources/Documentation/about.md]: Okay, then I would rephrase the sentence before the link to specifically say information about the Eiffel Gerrit plugin, instead of just Eiffel.

Comment (a-breid-ericsson) [reply src/main/resources/Documentation/about.md]: Doner.

File: src/main/resources/Documentation/about.md (hunk 8 above)
Comment (a-breid-ericsson) [reply src/main/resources/Documentation/about.md]: Done.











....
